exercises/grains/example.rb

```diff
@@ -1,9 +1,14 @@
+module BookKeeping
+  VERSION = 1
+end
+
 module Grains
   def self.square(number)
+    fail ArgumentError if number <= 0 || number > 64
```

Comment:
I had to make an adjustment to the implementation because the shared test data includes some error cases that weren't in the original ruby implementation.

How does this work with book keeping when it wasn't there originally? Normally, I'm assuming we would bump the `BookKeeping::VERSION`, but this exercise wasn't using book keeping yet.